daemon/control.c

```diff
@@ -1103,7 +1104,7 @@ control_handle_message(control_t *control, const ControllerToDaemon *msg, int fd
 	case CONTROLLER_TO_DAEMON__COMMAND__REMOVE_CONTAINER:
 		if (NULL == container) {
 			INFO("Container does not exist, nothing to destroy!");
-			control_send_message(CONTROL_RESPONSE_CMD_FAILED, fd);
```

Comment:
seems you implicitly reverted some changes from lately merged PRs.

Reply:
Done